src/sentry/workflow_engine/models/detector.py

```diff
@@ -21,17 +21,29 @@
 
 @region_silo_model
 class Detector(DefaultFieldsModel, OwnerModel):
-    __relocation_scope__ = RelocationScope.Organization
+    __relocation_scope__ = RelocationScope.Excluded
```

Comment:
Why'd we move to excluded instead of project?

Reply:
Hmm, I think we're just meant to use `Organization` actually like https://github.com/getsentry/sentry/blob/1c0cb149b011907a16882411a46c9b5181ca268e/src/sentry/models/options/project_option.py#L166-L175